src/cephadm/cephadm.py

```diff
@@ -3230,15 +3236,30 @@ def fetch_custom_config_files(ctx: CephadmContext) -> List[Dict[str, Any]]:
     return []
 
 
-def fetch_tcp_ports(ctx: CephadmContext) -> List[int]:
+def fetch_tcp_ports(ctx: CephadmContext) -> List[EndPoint]:
     """Return a list of tcp ports, as integers, stored on the given ctx.
```

Comment:
Hello, 

really just nitpicking, but the comment still mentions returning a list of TCP Ports as integers, but we return now a list of endpoints which is a list of EndPoint objects now